survey_placeholder/views/survey_question_views.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="survey_question_form" model="ir.ui.view">
+        <field name="model">survey.question</field>
+        <field name="inherit_id" ref="survey.survey_question_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[hasclass('oe_title')]" position="after">
```

Comment:
In 15 this class is applied in the title section only, so the field is being placed under the questions name, not inside the options tab: [](https://github.com/odoo/odoo/blob/15.0/addons/survey/views/survey_question_views.xml#L16C64-L16C64)

Reply:
Yes, the options in the question form are different in v16. So I decided to put it close to the question title which is quite related to the placeholder

Reply:
Ah, undestood. It looks correct there for me.